config/locales/en/en.yml

```diff
@@ -1,5 +1,9 @@
 en:
-  locale_name: English
+  locale_name:
+    en: English
+    es: Spanish
+    es-AR: Spanish (Argentina)
+    fr: French
```

Comment:
Not sure this is necessary. I think it's customary to list each language name in its own language. So English, Francais, Espanol, etc.

So perhaps just have the language's own name in its own file, then you can look up the others using I18n.l('locale_name', locale: :es)`